contrib/pkg/broker/server/server_test.go

```diff
@@ -23,17 +23,20 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	. "github.com/kubernetes-incubator/service-catalog/contrib/pkg/broker/server"
+	"github.com/kubernetes-incubator/service-catalog/contrib/pkg/broker/controller"
 	"github.com/kubernetes-incubator/service-catalog/pkg/brokerapi"
 )
 
 //
 // Test of server /v2/catalog endpoint.
 //
 
+// Make sure that Controller stub implements controller.Controller interface
+var _ controller.Controller = &Controller{}
```

Comment:
This line is redundant with the below calls to `createHandler`. Can you please remove it?